Formula/lit.rb

```diff
@@ -3,8 +3,8 @@ class Lit < Formula
 
   desc "Portable tool for LLVM- and Clang-style test suites"
   homepage "https://llvm.org"
-  url "https://files.pythonhosted.org/packages/75/e8/6deb4cc017266af4b8c5d370de340d917b0bbcbce3cbaf4ec1cfba7b0ca7/lit-16.0.3.tar.gz"
-  sha256 "25524fe51fa3261212cfd86a8891429ed0460e247384c5a2001612d08e362e00"
+  url "https://files.pythonhosted.org/packages/63/ab/e4c5f26400970a3ca356d19e592af3a5f05a73d5f35470accb6600f84045/lit-16.0.5.tar.gz"
```

Comment:
Kinda weird, since LLVM is only at 16.0.4. We should wait a bit in case upstream yanks this.